geomesa-raster/src/test/scala/org/locationtech/geomesa/raster/RasterTestsUtils.scala

```diff
@@ -123,4 +133,89 @@ object RasterTestsUtils {
 
   private def getDelta(level: Int): Double = 45.0 / Math.pow(2, level)
 
+  def compareIntBufferedImages(act: BufferedImage, exp: BufferedImage): Boolean = {
+    //compare basic info
+    if (act.getWidth != exp.getWidth) false
+    else if (act.getHeight != exp.getHeight) false
+    else {
+      val actWR = act.getRaster
+      val expWR = exp.getRaster
+      val actual = for(i <- (0 until act.getWidth).iterator;
```

Comment:
(Picky) You don't need the ;'s if you use {}'s instead.  Not sure if that works with parens.